### ISSUE TITLE
Causal masking of pre-trained BERT encoder

### ISSUE THREAD

**Issue by Hippogriff** [AnswerDotAI/ModernBERT#223]
I am looking for a pretrained BERT encoder that allows for causal masking, i.e. a token is not allowed to see the future tokens in the sequence. Can someone please point me to relevant script or resources?
